sdk/communication/azure-communication-rooms/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
 # Release History
 
-## 1.0.0-beta.1 (2022-08-10)
-This is the initial release of Azure Communication Services for rooms. For more information, please see the [README][read_me] and [documentation][documentation].
+## 1.0.0-beta.1 (2022-08-12)
+This is the initial release of Azure Communication Services for rooms. For more information, please see the [README](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/communication/azure-communication-rooms/README.md) and [documentation](https://docs.microsoft.com/en-us/azure/communication-services/concepts/rooms/room-concept).
```

Comment:
Build checks failed because of the `en-us` locale. Please replace with this:

"https://docs.microsoft.com/en-us/azure/communication-services/concepts/rooms/room-concept"

Reply:
Thanks for pushing the fix Radu.